Source/WebKit/UIProcess/API/APIHTTPCookieStore.cpp

```diff
@@ -116,10 +118,27 @@ void HTTPCookieStore::deleteCookie(const WebCore::Cookie& cookie, CompletionHand
 
 void HTTPCookieStore::deleteAllCookies(CompletionHandler<void()>&& completionHandler)
 {
-    if (auto* networkProcess = networkProcessIfExists())
-        networkProcess->sendWithAsyncReply(Messages::WebCookieManager::DeleteAllCookies(m_sessionID), WTFMove(completionHandler));
-    else
-        completionHandler();
+    auto callbackAggregator = CallbackAggregator::create(WTFMove(completionHandler));
+
+    if (m_owningDataStore) {
+        for (auto& processPool : m_owningDataStore->processPools()) {
+            processPool->forEachProcessForSession(m_sessionID, [&](auto& process) {
+                if (!process.canSendMessage())
+                    return;
+                auto pid = process.processIdentifier();
+                WTFLogAlways("CHRIS: sendWithAsyncReply WC BEGIN - PID: %d", pid);
```

Comment:
You probably want to take these out; there are multiple in more than once source file.

Reply:
Yes, sorry about this. I really miss webkit-patch's behavior that would show me a pretty-diff before uploading.

Reply:
I find that I always have to open a terminal window next to the window where I am composing my change log and carefully read the diff

But of course, I am doing that to compose my per-function comments, which apparently are not something other WebKit contributors want to require! That process is where I notice so many mistakes I have made